test/matlab_experimental/ctTestConstPressureReactor.m

```diff
@@ -0,0 +1,186 @@
+classdef ctTestConstPressureReactor < matlab.unittest.TestCase
+
+    properties
+        gas
+        gas1
+        gas2
+        r1
+        r2
+        solid
+        resGas
+        env
+        mfc1
+        mfc2
+        interface1
+        interface2
+        surf1
+        surf2
+        net1
+        net2
+    end
+
+    methods (TestClassSetup)
+
+        function testSetUp(self)
+            ctTestSetUp
+            copyfile('../data/testConstPressureReactor.yaml', ...
```

Comment:
Instead of copying files, we should set it up so that the `test/data` directory is part of the search path for input files.